gui/autodump.lua

```diff
@@ -331,7 +344,15 @@ function Autodump:do_dump(pos)
                 item.flags.forbid = true
             end
             if not on_ground then
-                dfhack.items.makeProjectile(item)
+                local proj = dfhack.items.makeProjectile(item)
+                if proj then
+                    proj.flags.no_impact_destroy = true
+                    proj.flags.bouncing = true
+                    proj.flags.piercing = true
+                    proj.flags.parabolic = true
+                    proj.flags.no_adv_pause = true
+                    proj.flags.no_collide = true
```

Comment:
should this be done inside of `makeProjectile`?

Reply:
This is correct for falling items, but not for all projectiles.

Reply:
ah ok